src/gt4py/next/program_processors/runners/dace_fieldview/gtir_sdfg.py

```diff
@@ -710,46 +710,51 @@ def _flatten_tuples(
             head_state.add_edge(src_node, None, nsdfg_node, connector, memlet)
 
         def make_temps(
-            output_data: gtir_builtin_translators.FieldopData,
+            inner_data: gtir_builtin_translators.FieldopData,
         ) -> gtir_builtin_translators.FieldopData:
             """
             This function will be called while traversing the result of the lambda
             dataflow to setup the intermediate data nodes in the parent SDFG and
             the data edges from the nested-SDFG output connectors.
             """
-            desc = output_data.dc_node.desc(nsdfg)
-            if desc.transient:
+            inner_desc = inner_data.dc_node.desc(nsdfg)
+            if inner_desc.transient:
                 # Transient nodes actually contain some result produced by the dataflow
                 # itself, therefore these nodes are changed to non-transient and an output
                 # edge will write the result from the nested-SDFG to a new intermediate
                 # data node in the parent context.
```

Comment:
Not part of the PR, but more correct would be saying something like "Although the data only exists within the SDFG, because it serves as an output of the nested SDFG it must be global" or something like that.
But feel free to ignore.